packages/graph/hash_graph/lib/graph/src/api/rest/mod.rs

```diff
@@ -4,7 +4,7 @@
 //! grouped router example
 
 mod api_resource;
-mod data_type;
+pub mod data_type;
```

Comment:
Just a quick question here, is this module used somehwer else for it to be `pub`?

Reply:
Discussed off-GitHub